tasks/recall-association.md

```diff
@@ -0,0 +1,52 @@
+# Summary
+> This study was designed to understand collaborative inhibition causes. Researchers compare individual recall (control) to two collaborative group conditions, 
+one in which people took turns recalling items together (See Group), the other in which people take turns recalling items but are not aware of the items their partner 
+has recalled (No See Group). Findings how that collaborative inhibition is due to interference caused by the product of recall- sharing answers with each other,
+not the process of taking turns without sharing. 
+
+
+# References
+>  Paper Link: https://link.springer.com/article/10.3758/BF03196740
+> Papers with Similar Tasks: 
+> Productivity Loss In Brainstorming Groups: Toward the Solution of a Riddle
+September 1987 Journal of Personality and Social Psychology 53(3):497-509
+DOI: 10.1037/0022-3514.53.3.497: https://www.researchgate.net/publication/232602288_Productivity_Loss_In_Brainstorming_Groups_Toward_the_Solution_of_a_Riddle
+
+> Group remembering: does social loafing underlie collaborative inhibition? M S Weldon 1, C Blair, P D Huebsch
+PMID: 11185783 DOI: 10.1037//0278-7393.26.6.1568
+
+
+# Stimuli
+> No Visuals Included in paper or appendix but booklet with words printed in black 20 point font was used 
```

Comment:
add, "with each word printed on a separate sheet of paper, displayed in the center"

Reply:
I see you added it later